src/docs/arch-index.dox

```diff
@@ -140,9 +139,10 @@ make up in-memory Btrees and subordinate data structures.
 
 A checkpoint is created by WiredTiger to serve as a point from which it can recover.
 
-@subpage arch-column
+@subpage arch-column-row
 
-Column Stores are Btrees that have as their key a record id.
+Row Stores and column store are B-Trees. Row Stores have a variable size key and data while column
+stores have as their key a record id.
```

Comment:
I think this is still in the wrong spot -- if you look at the main architecture page, all of the topic names are sorted except for this one. Not sure if that's necessary or not, but this is the only one that isn't sorted. (Well, the glossary at the end, I guess.)

Reply:
Ok, got it (I hope). I went back to what was suggested in the ticket and used the name `arch-row-column` instead, see 67f8e49092b73807708fc5f81bbf4081e681b5db.